### ISSUE TITLE
Android questions

### ISSUE THREAD

**Issue by KoleckOLP** [alexbatalov/fallout2-ce#182]
Hello I am trying the android build,
I can't tfigure out how to open the on screen keyboard to for example write a save file name.

Another thing I would like to propose is 2 finger tap for right click intead of tripple tap.

The android build seems to be really early in development right? 
So I assume there is no way to change resolution yet?

**Comment by bruenor41** [comment]
Yeah, just tried it today, I was too excited. It works, but controls on android really sucks. I have experiences with c++ and ndk for android, I can try to fix that, but someone must lead me a bit how to compile it
